OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/19.3.0/initsh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# LICENSE UPL 1.0
```

Comment:
This is not an approved header format for the UPL. Please use the [documented short reference](https://opensource.oracle.com/licenses/upl/#faq-7), i.e.:

```shell
# Copyright (c) 2022, Oracle and/or its affiliates
# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
```